scripts/src/javatests/org/oppia/android/scripts/coverage/CoverageReporterTest.kt

```diff
@@ -530,11 +630,55 @@ class CoverageReporterTest {
     CoverageReporter(
       tempFolder.root.absolutePath,
       coverageReportContainer,
-      ReportFormat.HTML
+      ReportFormat.HTML,
+      testExemptions
     ).generateRichTextReport()
 
     assertThat(outContent.toString().trim()).contains(
       "The file $exemptedFilePath is exempted from coverage analysis"
     )
   }
+
+  private fun readFinalMdReport(): String {
+    return File(
+      "${tempFolder.root}" +
+        "$coverageDir/CoverageReport.md"
+    ).readText()
+  }
+
+  private fun createTestFileExemptionTextProto():
+    Map<String, TestFileExemptions.TestFileExemption> {
```

Comment:
This should fit on one line--I suggest moving this up so that the method body can be indented like others in the class (for consistency).

Reply:
Having it in a single time caused max line length exceeded Lint check error or is there any other better way I can handle this?

Reply:
Ah, I thought it would fit. The other way is to directly import ``TestFileExemption`` so that it can be directly referenced instead of via ``TestFileExemptions``.

Reply:
Thanks, now I've adapted the approach followed with the #5480 to handle the exemptionList, and (removed the old implementation) now they fit right within the max length.

```
private fun createTestFileExemptionsProtoFile(testFileExemptions: TestFileExemptions): String {}
```